transformer/compose/utils.go

```diff
@@ -58,6 +72,257 @@ func IsV3(path string) (bool, error) {
 }
 */
 
+func createConfigMapName(sourcePath string) string {
+	cfgMapHashID := getHash([]byte(sourcePath))
+	cfgName := fmt.Sprintf("%s-%d", cfgMapPrefix, cfgMapHashID)
+	cfgName = common.MakeStringK8sServiceNameCompliant(cfgName)
+	return cfgName
+}
+
+func createSecretName(sourcePath string) string {
+	cfgMapHashID := getHash([]byte(sourcePath))
+	cfgName := fmt.Sprintf("%s-%d", secretPrefix, cfgMapHashID)
+	cfgName = common.MakeStringK8sServiceNameCompliant(cfgName)
+	return cfgName
+}
+
+func createVolumeName(sourcePath string, serviceName string) string {
+	hashID := getHash([]byte(sourcePath))
+	volumeName := fmt.Sprintf("%s-%s-%d", common.VolumePrefix, serviceName, hashID)
+	volumeName = common.MakeStringK8sServiceNameCompliant(volumeName)
+	return volumeName
+}
+
+func withinSizeLimit(filePath string) (bool, error) {
```

Comment:
probably rename it as `withinConfigMapSizeLimit`, so that it indicates what limit we are mentioning here.

Reply:
This is a common function for configmap and secrets. Shall I say rename it as `withinK8sMapSizeLimits`?